packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2816,7 +2828,7 @@ export type MessageThreadProps = {
     strings?: Partial<MessageThreadStrings>;
     fileDownloadHandler?: FileDownloadHandler;
     onDisplayDateTimeString?: (messageDate: Date) => string;
-    atMentionOptions?: AtMentionOptions;
```

Comment:
Just out of curiosity? Why do we change atMention to mention?

Reply:
> Just out of curiosity? Why do we change atMention to mention?

That's because we allow customization of the trigger. Contoso can set the trigger to anything so it might not be an @ sign. Since it can be $Mention or #Mention etc, we decided to go with Mention instead of atMention.